docs/source/tutorials/mint-your-first-asset.md

```diff
@@ -32,12 +32,12 @@ and may be easily modified if needed in future.
 
 ### 3. Use Iroha CLI Client to prepare Iroha Peer
```

Comment:
Duplicates tutorial. Prefer having pure tutorial and pure code repo. Replace with `Readme.md` which points to the tutorial github pages?

Reply:
I'll move this to the tutorial entirely in a separate pr